web-marketplace/src/hooks/useUpdateOrganizationProfile.ts

```diff
@@ -0,0 +1,265 @@
+                sanitizedDescription ?? currentConfig.description ?? '',
+              image_url:
+                sanitizedProfileImage ?? currentConfig.image_url ?? null,
+              automatically_add_cw20s:
+                currentConfig.automatically_add_cw20s ?? true,
+              automatically_add_cw721s:
+                currentConfig.automatically_add_cw721s ?? true,
+              dao_uri: currentConfig.dao_uri ?? null,
+            },
+          },
+        },
+      };
+
+      const actions: WasmExecuteAction[] = [updateConfigAction];
+
+      const maybeAddMetadataAction = (
+        key: 'banner' | 'website_link' | 'twitter_link',
+        nextValue: string | null,
+        previousValue?: string | null,
+      ) => {
+        const normalizedNext = nextValue ?? null;
+        let normalizedPrevious = previousValue ?? null;
+
+        if (key === 'banner') {
+          if (normalizedNext === DEFAULT_PROFILE_BG) {
+            normalizedPrevious = DEFAULT_PROFILE_BG;
+          }
+          if (normalizedPrevious === DEFAULT_PROFILE_BG) {
+            normalizedPrevious = null;
+          }
+        }
+
+        if (normalizedNext === normalizedPrevious) return;
+
+        if (normalizedNext) {
+          actions.push({
+            authorizationId,
+            roleId,
+            contract: daoAddress,
+            msg: {
+              set_item: {
+                key,
+                value: normalizedNext,
+              },
+            },
+          });
+          return;
+        }
+
+        if (normalizedPrevious) {
+          actions.push({
+            authorizationId,
+            roleId,
+            contract: daoAddress,
+            msg: {
+              remove_item: {
+                key,
+              },
+            },
+          });
+        }
+      };
+
+      maybeAddMetadataAction(
+        'banner',
+        sanitizedBackgroundImage,
+        currentValues?.backgroundImage ?? null,
+      );
+      maybeAddMetadataAction(
+        'website_link',
+        sanitizedWebsite,
+        currentValues?.websiteLink ?? null,
+      );
+      maybeAddMetadataAction(
+        'twitter_link',
+        sanitizedTwitter,
+        currentValues?.twitterLink ?? null,
+      );
+
+      const executeActionsMsg = getExecuteActionsWasm(actions);
+      const executeMsg = getMsgExecuteContract({
+        walletAddress: wallet.address,
+        contract: rbamAddress,
+        executeActionsMsg,
+      });
+
+      const result = await signAndBroadcast({
+        msgs: [executeMsg],
+        fee: 'auto',
+      });
+
+      if (typeof result === 'string') {
+        throw new Error(result);
+      }
+
+      // allow indexer to persist the latest RBAM metadata before refetching
+      await new Promise(resolve => setTimeout(resolve, 2000));
+
+      if (wallet?.address) {
+        await queryClient.invalidateQueries({
+          queryKey: getAccountByAddrQueryKey({ addr: wallet.address }),
+        });
+      }
```

Comment:
this is not needed since we're getting the org info from the AccountById query, not the AccountByAddr one